derive/src/lib.rs

```diff
@@ -1,16 +1,15 @@
 //! Ethereum (Solidity) derivation for rust contracts (compiled to wasm or otherwise)
 #![feature(alloc)]
-#![feature(proc_macro)]
+#![feature(use_extern_macros)]
 #![recursion_limit="128"]
+#![deny(unused)]
```

Comment:
it should be under `strict` flag, it might make development compilation uncomfortable

Reply:
I helped to spot two unused dependencies, among other other unused code. It's quite rare when you have an unused code (in most cases, it's some conditional compilation), and it's quite easy to add `allos(unused)` in suck cases. I propose to not move it under strict flag because it doesn't really make compilation uncomfortable.

I can do it if you're not convinced, but I'd recommend to give it a try at least. It always could be trimmed out later if you find yourself in such a situation.

Reply:
sure, this make sense!

but we usually do it like this:

https://github.com/paritytech/pwasm-std/blob/master/Cargo.toml#L31

and then:

https://github.com/paritytech/pwasm-std/blob/master/src/lib.rs#L5

so that when developing, it does not fire, only on travis

Reply:
Nice one, I'l probably steal it for my projects ;)

However, I still think it worth having `unused` by default. Denying on all warnings is makes you feel in fire (can't agree more on this!), but having enabled `unused` is not this disturbing.